src/reaper_osara.cpp

```diff
@@ -1242,7 +1242,11 @@ bool maybeSwitchToFxPluginWindow() {
 // Handle keyboard keys which can't be bound to actions.
 // REAPER's "accelerator" hook isn't enough because it doesn't get called in some windows.
 LRESULT CALLBACK keyboardHookProc(int code, WPARAM wParam, LPARAM lParam) {
-	if (code != HC_ACTION && wParam != VK_APPS && wParam != VK_RETURN && wParam != VK_F6) {
```

Comment:
Oops. The `code` check probably should have been an or, not an and. Anyway, the way you've done it is probably easier to read.